test/test_admin.py

```diff
@@ -23,30 +22,37 @@ def admin(conn_string):
 
         # alternatively we could warn somewhere (README, when running the tests, or other places)
         # that this are not intended to run against any other stardog deployment that is not a clean one.
-        for db in admin.databases():
-            db.drop()
-
-        for user in admin.users():
-            if user.name not in DEFAULT_USERS:
-                user.delete()
-
-        for role in admin.roles():
-            if role.name not in DEFAULT_ROLES:
-                role.delete()
-
-        for stored_query in admin.stored_queries():
-            stored_query.delete()
-
-        for vg in admin.virtual_graphs():
```

Comment:
removing VG and data sources as I discovered a bug in which you can't delete initialized VGs: https://github.com/stardog-union/pystardog/issues/60